src/main/java/com/hubspot/jinjava/lib/exptest/IsFloatExpTest.java

```diff
@@ -0,0 +1,30 @@
+package com.hubspot.jinjava.lib.exptest;
+
+import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
+import com.hubspot.jinjava.doc.annotations.JinjavaParam;
+import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.interpret.JinjavaInterpreter;
+
+@JinjavaDoc(
+  value = "Return true if object is a float",
+  input = @JinjavaParam(value = "value", type = "object", required = true),
+  snippets = {
+    @JinjavaSnippet(
+      code = "{% if num is float %}\n" +
+      "      <!--code to render if num contains an floating point value-->\n" +
+      "{% endif %}"
+    )
+  }
+)
+public class IsFloatExpTest implements ExpTest {
+
+  @Override
+  public String getName() {
+    return "float";
+  }
+
+  @Override
+  public boolean evaluate(Object var, JinjavaInterpreter interpreter, Object... args) {
+    return var instanceof Double || var instanceof Float;
```

Comment:
I think we need to check `BigDecimal` as well (see the `AddFilter`). BigDecimal support was added in the math filters since python supports arbitrary precision and we had to come up with a workaround for Java.